bug_view_inc.php

```diff
@@ -946,16 +946,16 @@ function bug_view_relationship_view_box( $p_bug_id, $p_can_update ) {
 	}
 
 	$t_relationship_graph = ON == config_get( 'relationship_graph_enable' );
-	$t_show_top_div = $p_can_update || $t_relationship_graph;
-	?>
+	$t_event_buttons = event_signal( 'EVENT_MENU_ISSUE_RELATIONSHIP', $p_bug_id );
```

Comment:
I would sent the parameter to the event as follows: `issue_id` => $p_bug_id, rather than use order of parameters.  This follows the new model we are trying to adopt.

Reply:
I did it that way to be consistent with similar event EVENT_MENU_ISSUE, but I'm fine to change it.

Reply:
I would rather than we have all new events follow the new model.

Reply:
Follow-up in https://github.com/mantisbt/mantisbt/pull/1567#issuecomment-561543366